cloudlink-combined-storage/pom.xml

```diff
@@ -0,0 +1,197 @@
+            <version>${attach.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.gluonhq.attach</groupId>
+            <artifactId>storage</artifactId>
+            <version>${attach.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.gluonhq.attach</groupId>
+            <artifactId>util</artifactId>
+            <version>${attach.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish</groupId>
+            <artifactId>jakarta.json</artifactId>
+            <version>1.1.5</version>
+            <classifier>module</classifier>
+            <scope>runtime</scope>
+        </dependency>
+        <dependency>
+            <groupId>org.glassfish.jaxb</groupId>
+            <artifactId>jaxb-runtime</artifactId>
+            <version>2.3.1</version>
+            <scope>runtime</scope>
+        </dependency>
+    </dependencies>
+
+    <build>
+        <plugins>
+            <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.8.1</version>
+            </plugin>
+
+            <plugin>
+                <groupId>org.openjfx</groupId>
+                <artifactId>javafx-maven-plugin</artifactId>
+                <version>${javafx.plugin.version}</version>
+                <configuration>
+                    <mainClass>${main.class}</mainClass>
+                </configuration>
+            </plugin>
+
+            <plugin>
+                <groupId>com.gluonhq</groupId>
+                <artifactId>client-maven-plugin</artifactId>
+                <version>${client.plugin.version}</version>
+                <configuration>
+                    <mainClass>${main.class}</mainClass>
+                    <target>${client.target}</target>
+                    <verbose>true</verbose>
+                    <attachList>
+                        <list>display</list>
+                        <list>lifecycle</list>
+                        <list>statusbar</list>
+                        <list>storage</list>
+                    </attachList>
+                    <reflectionList>
+                        <list>javafx.fxml.FXMLLoader</list>
+                        <list>com.gluonhq.combinedstorage.model.Model</list>
+                        <list>com.gluonhq.combinedstorage.model.Note</list>
+                        <list>com.gluonhq.combinedstorage.service.Service</list>
+                        <list>com.gluonhq.combinedstorage.views.EditionPresenter</list>
+                        <list>com.gluonhq.combinedstorage.views.NotesPresenter</list>
+                    </reflectionList>
+                    <bundlesList>
+                        <list>com.gluonhq.combinedstorage.views.edition</list>
+                        <list>com.gluonhq.combinedstorage.views.notes</list>
+                    </bundlesList>
```

Comment:
`org.glassfish.json.messages` is required too

Reply:
Is this causing an issue on your end?

Reply:
I can reproduce this without the cloudlink config file.

Reply:
Yes, that was the case, I tested first without config file.